### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by RvanB**
Text updates!

### CODE REVIEW

File: client/components/public/FAQ.jsx [hunk 1]
```diff
@@ -259,9 +261,11 @@ FAQ = class FAQ extends Component {
 
             Schedule for the day:
             {schedule}
+            {/*
```
Comment (cabeese) [inline client/components/public/FAQ.jsx]: `virtualeventonly`?

File: client/components/public/FAQ.jsx [hunk 2]
```diff
@@ -386,17 +390,23 @@ FAQ = class FAQ extends Component {
...
-              <List.Item description="At least one smart phone. The more the better! (Think about battery life). Googling is encouraged!"/>
-              <List.Item description="A clip board, or a note pad. Graph paper might come in handy."/>
-              <List.Item description="Scissors and tape."/>
-              <List.Item description="Writing utensils (pencils, erasers, highlighters)."/>
-              <List.Item description="A felt tip marking pen - something that will mark a folded object, but not crush the object."/>
-              <List.Item description="Standard 10-12 pack of colored pencils."/>
-              <List.Item description="Water to drink and a sack lunch or snacks! Light refreshments will be provided."/>
-              <List.Item description="Umbrella to stay dry!"/>
-              <List.Item description="Hole punch for your puzzles." />
+            <List bulleted>
```
Comment (cabeese) [inline client/components/public/FAQ.jsx:393]: I'm assuming the only thing changed here is everything is wrapped in the `<List />` and indented, so not reading very closely

**Review by cabeese — APPROVED** [review]

File: client/components/public/FAQ.jsx (hunk 2 above)
Comment (RvanB) [reply client/components/public/FAQ.jsx:393]: Yeah it seemed like quite a few if not all the bullets were slightly reworded or completely different so I just rewrote all items of the list!

File: client/components/public/FAQ.jsx (hunk 1 above)
Comment (RvanB) [reply client/components/public/FAQ.jsx]: Fixed!

**Pull request merged**